tools/import-mbox.py

```diff
@@ -166,7 +166,7 @@ def printid(self, message):
         print("%s: %s" % (self.name, message))
 
     def run(self):
-        global goodies, baddies, dedupped
+        global goodies, baddies, dedupped, es
```

Comment:
If `es` isn't declared as a global here then using `--dedup` without `--dry` breaks because the `es = ...` assignment on L346 makes it a local.